sdk/lusid/api_client.py

```diff
@@ -4,7 +4,7 @@
 
     FINBOURNE Technology  # noqa: E501
 
-    The version of the OpenAPI document: 0.11.3725
+    The version of the OpenAPI document: 0.11.3704
```

Comment:
Ignoring this file as discussion is here: https://github.com/finbourne/lusid-sdk-generators/pull/67/files#diff-fc4906a4ca399ef1ad7bc41cc8c280be502bafde4f0868c9e21e55951cc158f3R647-R648